### PR TITLE
Newbranch5

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

File: bla-1.php [hunk 1]
```diff
@@ -11,3 +11,18 @@
 );
 
 echo esc_html( print_r($a, 1) );
```
Comment (gudmdharalds-a8c) [inline bla-1.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

File: bla-1.php [hunk 2]
```diff
@@ -11,3 +11,18 @@
...
+
+
+
+$m = time() . get_term_by( 0, 1 );
```
Comment (gudmdharalds-a8c) [inline bla-1.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-1.php [hunk 3]
```diff
@@ -11,3 +11,18 @@
...
+print_r( $m );
```
Comment (gudmdharalds-a8c) [inline bla-1.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

File: bla-1.php [hunk 4]
```diff
@@ -11,3 +11,18 @@
...
+
+$a = new WP_Query(
+	array(
+		'number_posts' => -1
+	)
+);
+
+echo esc_html( print_r($a, 1) );
```
Comment (gudmdharalds-a8c) [inline bla-1.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

File: bla-2.php [hunk 5]
```diff
@@ -31,4 +31,6 @@
 
 echo time() . ':      ' . get_term_by( 3, 4 );
```
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-2.php [hunk 6]
```diff
@@ -31,4 +31,6 @@
...
 
+echo time() . ':      ' . get_term_by( 3, 4 );
```
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-2.php (hunk 5 above)
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-2.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-2.php (hunk 5 above)
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-2.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-2.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-3.php [hunk 7]
```diff
@@ -8,4 +8,9 @@
 	)
 );
 
-echo print_r($a, 1)
+echo print_r($a, 1);
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 8]
```diff
@@ -8,4 +8,9 @@
...
+
+
+echo time() . print_r($a, 1);
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 9]
```diff
@@ -8,4 +8,44 @@
 	)
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 10]
```diff
@@ -8,4 +8,63 @@
...
 );
 
-echo print_r($a, 1)
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 11]
```diff
@@ -8,4 +8,64 @@
 	)
 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 12]
```diff
@@ -8,4 +8,64 @@
...
 
-echo print_r($a, 1)
+echo print_r($a, 1);
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 13]
```diff
@@ -8,4 +8,66 @@
 	)
 );
 
-echo print_r($a, 1)
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 14]
```diff
@@ -8,4 +8,69 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 15]
```diff
@@ -1,11 +1,80 @@
 <?php
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'get_term_by'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: get_term_by() is prohibited, please use wpcom_vip_get_term_by() instead.

File: bla-3.php [hunk 16]
```diff
@@ -1,11 +1,80 @@
...
 
 $a = new WP_Query(
 	array(
 		'number_posts' => -1
 	)
 );
 
-echo print_r($a, 1)
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 17]
```diff
@@ -1,11 +1,80 @@
...
+echo print_r($a, 1);
+
+
+echo time() . print_r($a, 1);
+
+
+
+
+
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 18]
```diff
@@ -1,11 +1,80 @@
...
+
+
+
+
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 19]
```diff
@@ -1,11 +1,84 @@
 <?php
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 20]
```diff
@@ -1,11 +1,84 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 21]
```diff
@@ -1,11 +1,84 @@
...
 
 $a = new WP_Query(
 	array(
 		'number_posts' => -1
 	)
 );
 
-echo print_r($a, 1)
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 22]
```diff
@@ -1,11 +1,84 @@
...
+echo print_r($a, 1);
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 23]
```diff
@@ -1,11 +1,84 @@
...
+
+
+echo time() . print_r($a, 1);
+echo time() . print_r($a, 1);
+
+
+
+
+
+
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 24]
```diff
@@ -1,11 +1,84 @@
...
+
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 25]
```diff
@@ -1,11 +1,86 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 26]
```diff
@@ -1,11 +1,86 @@
...
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 27]
```diff
@@ -1,11 +1,86 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 28]
```diff
@@ -1,11 +1,98 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 29]
```diff
@@ -1,11 +1,98 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 30]
```diff
@@ -1,11 +1,98 @@
...
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 31]
```diff
@@ -1,11 +1,98 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 32]
```diff
@@ -1,11 +1,98 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 33]
```diff
@@ -1,11 +1,98 @@
...
 
 $a = new WP_Query(
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 34]
```diff
@@ -1,11 +1,98 @@
...
 	array(
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 35]
```diff
@@ -1,11 +1,102 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 36]
```diff
@@ -1,11 +1,102 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 37]
```diff
@@ -1,11 +1,102 @@
...
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 38]
```diff
@@ -1,11 +1,102 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 39]
```diff
@@ -1,11 +1,92 @@
...
 $a = new WP_Query(
 	array(
 		'number_posts' => -1
 	)
 );
 
-echo print_r($a, 1)
+echo print_r($a, 1);
+
+
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 40]
```diff
@@ -1,11 +1,96 @@
 <?php
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 41]
```diff
@@ -1,11 +1,96 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 42]
```diff
@@ -1,11 +1,96 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 43]
```diff
@@ -1,11 +1,94 @@
...
 
 $a = new WP_Query(
 	array(
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: print_r() found. Debug code should not normally be used in production.

Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'print_r'

File: bla-3.php [hunk 44]
```diff
@@ -1,11 +1,95 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 45]
```diff
@@ -1,11 +1,95 @@
...
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 46]
```diff
@@ -1,11 +1,101 @@
 <?php
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 47]
```diff
@@ -1,11 +1,101 @@
...
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 48]
```diff
@@ -1,11 +1,101 @@
...
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
 
 $a = new WP_Query(
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Warning**: Found: ==. Use strict comparisons (=== or !==).

File: bla-5.php [hunk 49]
```diff
@@ -19,6 +19,8 @@
 
 
 
+echo "time4444: " . time() . '-' . get_term_by( 1, 2 );
+
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: **Warning**: Found: ==. Use strict comparisons (=== or !==).

File: bla-3.php [hunk 50]
```diff
@@ -1,11 +1,101 @@
 <?php
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 51]
```diff
@@ -1,11 +1,101 @@
...
 
 $a = new WP_Query(
 	array(
 		'number_posts' => -1
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php (hunk 38 above)
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-3.php [hunk 52]
```diff
@@ -1,11 +1,102 @@
 <?php
 
-echo "jamms: " . time() . '-' . get_term_by( 1, 2 );
+echo "jamms: " . time() . '-' . get_term_by( 1, 2 ) . 'bla';
 
 $a = new WP_Query(
```
Comment (gudmdharalds-a8c) [inline bla-3.php]: **Error**: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'time'

File: bla-5.php [hunk 53]
```diff
@@ -19,6 +19,8 @@
...
```
Comment (gudmdharalds-a8c) [inline bla-5.php]: **Warning**: Found: ==. Use strict comparisons (=== or !==).

**Comment by gudmdharalds** [comment]
test

**Comment by gudmdharalds** [comment]
test

**Pull request merged**